group_vars/all/rpc-o.yml

```diff
@@ -30,10 +30,21 @@ horizon_custom_uploads:
 repo_build_pip_extra_indexes:
   - "https://rpc-repo.rackspace.com/pools"
 
-# Tempest test execution options
-tempest_run_tempest_opts:
-  - "--serial"
-# Tempest testr options
-tempest_testr_opts: []
+# Elasticsearch versions
+# NOTE (alextricity25):
+# The elasticsearch version variables needs to be available to the repo_build role
+# due to the way it builds out the distro packages dataset. Distro packages within
+# a role are indexed using their raw form(jinja is not rendered). The repo_build
+# role then attempts to render these datasets, and will fail if it cannot resolve
+# a variable. Putting these variables in here ensures the repo_build
+# role can interpolate the variables correctly.
+elasticsearch_version: 2.4.1
+elasticsearch_reindex_version: 1.7.5
+
 # Tempest tests to run. A one-line, space-delimited string
-tempest_test_sets: "scenario defcore cinder_backup"
+tempest_test_whitelist:
+  - "tempest\\.scenario\\.test_(minimum|swift|server|dashboard)_basic(_ops)?"
```

Comment:
Good point. I was just looking at the defaults defined [here](https://github.com/openstack/openstack-ansible-os_tempest/blob/stable/ocata/defaults/main.yml#L59-L66), and it doesn't seem like we need to change anything to include all of the scenario tests that we need. I'll remove this override.